extension/src/format/sexpression.ts

```diff
@@ -1105,7 +1100,7 @@ export class Sexpression extends LispAtom {
                     else if (opName == "setq") {
                         return exp.formatSetq(startColumn);
                     }
-                    else if (opName == "foreach") {
+                    else if (opName === "foreach" || opName === "vlax-for") {
```

Comment:
I am thinking that maybe this is naive fix here. In my memory I remember that in OLD vlide there is a setting for if the first word is longer than 9 chars, it should use NARROW formating style, otherwise it should use the WIDE formatting style.